optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -1078,8 +1082,32 @@
   &lt;/solverBenchmark&gt;
 &lt;/#list&gt;
 &lt;/#list&gt;
+
+&lt;#list [1, 100, 1000] as acceptedCountLimit&gt;
+&lt;#list [1hard/5medium/10soft, 1hard/10medium/20soft, 1hard/20medium/50soft, 1hard/50medium/70soft] as startingTemperature&gt;
+  &lt;solverBenchmark&gt;
+    &lt;name&gt;SimulatedAnnealing acceptedCountLimit ${acceptedCountLimit} temperature ${startingTemperature?replace("/", "_")}&lt;/name&gt;
+    &lt;solver&gt;
+      &lt;localSearch&gt;
+        &lt;acceptor&gt;
+          &lt;simulatedAnnealingStartingTemperature&gt;${startingTemperature?replace("/", "_")}&lt;/simulatedAnnealingStartingTemperature&gt;
+        &lt;/acceptor&gt;
+        &lt;forager&gt;
+          &lt;acceptedCountLimit&gt;${acceptedCountLimit}&lt;/acceptedCountLimit&gt;
+        &lt;/forager&gt;
+      &lt;/localSearch&gt;
+    &lt;/solver&gt;
+  &lt;/solverBenchmark&gt;
+&lt;/#list&gt;
+&lt;/#list&gt;
 &lt;/plannerBenchmark&gt;</programlisting>
 
+      <note>
+        <para>When using Simulated Annealing Matrix Benchmarking on Windows it is necessary to replace the "/" from the
+        <literal>startingTemperature</literal> (<literal>?replace("/", "_")</literal>) so that it is not part of the
+        file name. Otherwise there is a problem with the naming conventions of files on Windows.</para>
```

Comment:
It's not just on windows: SolverBenchmark's name is validated against an regex that doesn't allow /, so it fails the same on all platforms.

I 'll merge and adjust that :)